### PR TITLE
Add an example getting IPv4 addresses from NetworkManager via dbus

### PR DESCRIPTION
**Pull request by mtoman**

### CODE REVIEW

File: shared/network/2.x.md [hunk 1]
```diff
@@ -362,6 +362,55 @@ settings.AddConnection(con)
...
+        addresses.append("%s/%s" % (addr.compressed, address[1]))
+
+    # Get device name
+    device_name = device_props.Get("org.freedesktop.NetworkManager.Device", "Interface")
+
+    # Print all the information for a single device
+    print("%s: %s" % (device_name, ", ".join(addresses)))
+```
+
+The above example will print device name and all associated IPv4 addresses for each network device with an IPv4 connection.
```
Comment (vipulgupta2048) [inline shared/network/2.x.md:412]: The above line might be better off as the one being before the code snippet to clearly explain what the code snippet is all about. This way folks know what they are looking at before scrolling further down and looking for explanations.

Comment (vipulgupta2048) [reply shared/network/2.x.md:412]: I would say the same for the example above it, as having a description before the example provides much needed context to the code in the example.

**Review by vipulgupta2048 — COMMENTED** [review]
@mtoman reviews are inline, otherwise LGTM. Merge when ready.

**Comment by mtoman** [comment]
@balena-ci rebase

File: shared/network/2.x.md (hunk 1 above)
Comment (mtoman) [reply shared/network/2.x.md]: Moved both descriptions above

**Comment by mtoman** [comment]
I've been encouraged to merge, therefore

**Comment by mtoman** [comment]
@balena-ci I self-certify!

**Pull request merged**